assets/js/dist/fade.js

```diff
@@ -1,40 +1,38 @@
 "use strict";
 
-$(document).ready(function () {
-  var mobileOffset = 270; // Start with only work section showing
+var mobileOffset = 270; // Start with nav-work button highlighted and only work section showing
 
-  $('#nav-work').removeClass('default-btn').addClass('current-btn');
-  $('#work').addClass('display');
-  var elements = document.getElementsByClassName('nav-btn');
-  Array.from(elements).forEach(function (el) {
-    el.addEventListener('click', function (e) {
-      e.preventDefault();
-      $('li').removeClass('current-btn');
-      $("#".concat(e.target.id)).addClass('current-btn');
-      $('#fade-wrapper').fadeOut(500, function () {
-        $('section').removeClass('display');
-        var sectionId = e.target.id.slice(4);
-        $("#".concat(sectionId)).addClass('display');
+document.getElementById('nav-work').classList.add('current-btn');
+document.getElementById('work').classList.add('display');
+var navElements = document.getElementsByClassName('nav-btn');
+Array.from(navElements).forEach(function (navElement) {
+  navElement.addEventListener('click', function (e) {
+    e.preventDefault();
+    Array.from(document.getElementsByTagName('li')).forEach(function (el) {
+      return el.classList.remove('current-btn');
+    });
+    document.getElementById(e.target.id).classList.add('current-btn');
+    $('#fade-wrapper').fadeOut(500, function () {
+      Array.from(document.getElementsByTagName('section')).forEach(function (el) {
+        return el.classList.remove('display');
+      });
+      document.getElementById(e.target.id.slice(4)).classList.add('display');
```

Comment:
To me this `slice` call is a [code smell](https://en.wikipedia.org/wiki/Code_smell). Ids should be just ids. If you need to share additional information, maybe consider a data attribute? Think about the case where some other engineer (or you in the future) want to change this id. How would you know that this fade file has an implicit dependency on the current naming scheme?